lib/child_process.js

```diff
@@ -140,9 +140,21 @@ exports.exec = function(command /*, options, callback*/) {
                           opts.callback);
 };
 
-Object.defineProperty(exports.exec, customPromisifyArgs,
-                      { value: ['stdout', 'stderr'], enumerable: false });
-
+Object.defineProperty(exports.exec, util.promisify.custom, {
+  enumerable: false, value: function(...args) {
+    const promise = createPromise();
+    exports.exec.call(this, ...args, (err, stdout, stderr) => {
```

Comment:
Definitely, that was my first all. And to use an arrow function instead of a regular function (as the property value). But then I looked at `util.promisify` and that's what it does, so I got all afraid that in some cases the `this` in exec may have some meaning.

OK - I'll refactor the value of the property into an arrow function + call `exec` regulary without `.call`.